apps/pwabuilder-vscode/src/services/usage-analytics.ts

```diff
@@ -1,6 +1,19 @@
 import { setup, defaultClient } from 'applicationinsights';
 import { getFlag } from '../flags';
 
+import * as vscode from 'vscode';
+import { Headers } from 'node-fetch';
+
+const sessionID = getSessionID();
+export const standard_headers = new Headers(
+  { 
+    "content-type": "application/json",
+    "Platform-Identifier": "PWAStudio",
+    "Platform-Identifier-Version": "",
```

Comment:
You could avoid passing the version at all if you're not tracking it, otherwise it will show up as a non-null empty string in the db.